lib/process-worker.js

```diff
@@ -166,11 +166,7 @@ process.on('message', async (message) => {
         console.log('Stop capturing artifacts sessionId', sessionId);
       }
 
-      if (message.test.status.toLowerCase() === 'pending') {
-        zebrunnerApiClient.deleteTest(message.test).then(() => {
```

Comment:
please, return any uniqie temporary log message. I'd like to see full logs if we are inside this if for smoke and full acceptance